src/sync/pull-request.ts

```diff
@@ -46,7 +46,7 @@ export default async function(
 ) {
 	logger.info("Syncing PRs: started");
 	let status: number;
-	let headers: Headers = {};
```

Comment:
Why this change?

Reply:
Forgot to mention. With update of the library the Headers interface become incompatible with the type of the headers from the AxiosResponse.

Do you have any ideas on how to make it better?

Reply:
Change the `Headers` interface to extend `AxiosResponseHeaders`?